zio-http/src/main/scala/zio/http/model/Request.scala

```diff
@@ -1,4 +1,4 @@
-package zio.http
+package zio.http.model
 
 import io.netty.channel.ChannelHandlerContext
 import io.netty.handler.codec.http.{DefaultFullHttpRequest, FullHttpRequest, HttpRequest}
```

Comment:
I think the import of the `service` package in line 7 is odd.

Reply:
- `ServerInboundHandler` is imported for encoding `Body`, as I wrote there it is not a pure data structure, here we are directly translating streaming bodies into netty context
- Note that `Ctx` is a type alias for `ChannelHandlerContext` and both are used in this file, that could be cleaned up
- Capturing and exposing the `ChannelHandlerContext` in `Request` is also a bit ugly but I think right now it is the only way to access the context from the `Http` level. I think it is only used for encoding streaming bodies (mentioned above) and to be able to get remote address but it is also exposed on the API so it is possible that there are other uses. Anyway I think changing this is out of the scope of this PR